rpc_jobs/standard_job_postmerge.yml

```diff
@@ -0,0 +1,141 @@
+- project:
+    name:       "gating-post-merge"
+    repo_url:   "https://github.com/rcbops/rpc-gating"
+    branch:     "master"
+    image:      "xenial"
+    scenario:   "functional"
+    action:     "test"
+    jobs:
+      - 'PM_{name}-{branch}-{image}-{scenario}-{action}'
+
+- job-template:
+    name: 'PM_{name}-{branch}-{image}-{scenario}-{action}'
+    project-type: pipeline
+    concurrent: true
+    FLAVOR: "performance1-1"
+    IMAGE: "Ubuntu 16.04 LTS (Xenial Xerus) (PVHVM)"
+    properties:
+      - build-discarder:
+          num-to-keep: "30"
+      - github:
+          url: "{repo_url}"
+    parameters:
+      - rpc_gating_params
+      - instance_params:
+          IMAGE: "{IMAGE}"
+          FLAVOR: "{FLAVOR}"
+          REGIONS: "{REGIONS}"
+          FALLBACK_REGIONS: "{FALLBACK_REGIONS}"
+      - string:
+          name: REPO_URL
+          default: "{repo_url}"
+          description: Url of the repo under test
+      - string:
+          name: BRANCH
+          default: "{branch}"
+          description: Branch of the repo under test
+      - string:
+          name: STAGES
+          default: >-
+            Allocate Resources,
+            Connect Slave,
+            Cleanup,
+            Destroy Slave
+          description: |
+            Pipeline stages to run CSV. Note that this list does not influence execution order.
+            Options:
+              Allocate Resources
+              Connect Slave
+              Cleanup
+              Destroy Slave
+    triggers:
+      - timed: "@daily"
+      - github
+
+    dsl: |
+      library "rpc-gating@${{RPC_GATING_BRANCH}}"
+      properties([pipelineTriggers([githubPush()])])
+
+      // Pass details about the job parameters through
+      // to the target environment so that scripts can
+      // use them to adapt behaviour.
+      env.RE_JOB_NAME = "{name}"
+      env.RE_JOB_IMAGE = "{image}"
+      env.RE_JOB_SCENARIO = "{scenario}"
+      env.RE_JOB_ACTION = "{action}"
+      env.RE_JOB_FLAVOR = "{FLAVOR}"
+      env.RE_JOB_TRIGGER = "periodic"
```

Comment:
Added

Reply:
Why ``periodic`` instead of ``PM`` (ie post merge)? Basically I'm wondering why the job name and this value do not match and what happens when a PM test is implemented which is triggered by a merge/tag?

Reply:
Changed RE_JOB_TRIGGER to PM.